doc/src/gotchas.rst

```diff
@@ -784,3 +784,63 @@ from SymPy 0.6.5:
         """
         expr = Poly.cancel(powsimp(expr))
         return powsimp(together(expr.expand()), combine='exp', deep=True)
+
+
+
+Be careful about substituting for derivatives! 
+==============================================
+.. code-block:: python
+
+    In [1]: import sympy as sm
+            import sympy.physics.mechanics as
+    In [2]: q1, q2 = me.dynamicsymbols('q1, q2')
+    In [3]: a, b, c = sm.symbols('a, b, c')
+    In [4]: expr = q1.diff().diff()**2*q1 + q1.diff()*q2.diff()
+            expr
```

Comment:
Use normal Python blocks here, not IPython. That way it will be doctested.

Reply:
Okay. Thanks for reviewing.